vm/src/builtins/dict.rs

```diff
@@ -833,3 +833,18 @@ impl PyMapping {
         self.dict
     }
 }
+
+impl<K, V> TryFromObject for std::collections::HashMap<K, V>
```

Comment:
is HashMap really nessessary? If we just need a map, dict internal is a sort of ordered map

Reply:
That was my change from the `RustPython:_sre-wip` branch, I think... I definitely agree now that it's not necessary, I think I was just trying to get something vaguely working